tests/test_msg_ethereum_sablier.py

```diff
@@ -0,0 +1,59 @@
+# This file is part of the KEEPKEY project.
+#
+# Copyright (C) 2021 Shapeshift
+#
+# This library is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this library.  If not, see <http://www.gnu.org/licenses/>.
+
+from base64 import b64encode
+import unittest
+import common
+import binascii
+import struct
+
+import keepkeylib.messages_pb2 as proto
+import keepkeylib.types_pb2 as proto_types
+import keepkeylib.exchange_pb2 as proto_exchange
+from keepkeylib.client import CallException
+from keepkeylib.tools import int_to_big_endian
+
+class TestMsgEthereumSablier(common.KeepKeyTest):
+    
+    def test_sign_salarywithdrawl(self):
```

Comment:
Is `salarywithdrawl` instead of `salarywithdrawal` a typo, or is the spelling intentional?

Reply:
typo!